langchain/chains/api/base.py

```diff
@@ -61,16 +65,18 @@ def validate_api_answer_prompt(cls, values: Dict) -> Dict:
             )
         return values
 
-    def _call(self, inputs: Dict[str, str]) -> Dict[str, str]:
+    def _call(
+        self,
+        inputs: Dict[str, str],
+        run_manager: CallbackManagerForChainRun = NullCallbackManagerForChainRun(),
```

Comment:
could maybe refactor base __call__ so that this doesn't even have to passed in as kwarg?